src/general/DayNightConfiguration.cs

```diff
@@ -0,0 +1,30 @@
+﻿using Newtonsoft.Json;
+
+/// <summary>
+///   Definition for the day night cycle.
+/// </summary>
+/// <remarks>Values for the DayNightCycle, as given in day_night_cycle.json</remarks>
+public class DayNightConfiguration : IRegistryType
+{
+    public string InternalName { get; set; } = null!;
+
+    /// <summary>
+    ///   Number of in-game hours per in-game day.
+    /// </summary>
+    [JsonProperty]
+    public float HoursPerDay { get; private set; }
+
+    /// <summary>
+    ///   Percentage of the in-game day which has sunlight.
+    /// </summary>
+    [JsonProperty]
+    public float DaytimePercentage { get; private set; }
```

Comment:
Is this a percentage or a fraction? Most of the game systems use fractions to be easier for computing. I'd very highly recommended that this is switched to be a fraction rather than a percentage.

Reply:
It is a fraction (if by that you mean a value between 0 and 1). Probably warrants a name change.

Reply:
What would that name change be though? I think DaytimeFraction is even more confusing. Maybe the value should just be noted in the summary?

Reply:
Everywhere else in the game I've enforced the convention that variables with "fraction" in the name mean the valid range is 0-1 and anything with "percentage" in the name has valid value range of 0-100. I'll make a change to the styleguide about this naming in my launcher 2.0 features branch.